styles/components/_buttons.scss

```diff
@@ -325,6 +325,30 @@
   }
 }
 
+.btnFlx {
+  // see styleSheet.html for example usage.
+  // use for buttons in the new 'flex' style with more padding and use css flex.
+  @extend .btn;
+  padding: 25px;
+
+  border-radius: 0px;
+  border-top: 1px solid;
+
+  & + a {
+    border-left: 1px solid #dbdbdb; // Move this color into themes.
+  }
+
+  + a:hover {
```

Comment:
This is selecting a sibling to the right when the sibling is hovered, isn't it?

I think you want 
```
&:hover + .btnFlx
```